parlai/core/torch_classifier_agent.py

```diff
@@ -286,8 +285,8 @@ def eval_step(self, batch):
         else:
             labels = self._get_labels(batch)
             loss = self.criterion(scores, labels)
-            self.metrics['loss'] += loss.item()
-            self.metrics['examples'] += len(batch.text_vec)
+            self.record_local_metric('loss', AverageMetric.many(loss))
```

Comment:
related to above comment, i thought part of the reason we were calling `.item()` was to get the scalar value rather than keep the tensor around

Reply:
Metrics automatically calls item